component/prometheus/integration/process_exporter/process_exporter.go

```diff
@@ -0,0 +1,79 @@
+package process_exporter
+
+import (
+	"github.com/grafana/agent/component"
+	"github.com/grafana/agent/component/prometheus/integration"
+	"github.com/grafana/agent/pkg/integrations"
+	"github.com/grafana/agent/pkg/integrations/process_exporter"
+	exporter_config "github.com/ncabatoff/process-exporter/config"
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name:    "prometheus.integration.process_exporter",
+		Args:    Config{},
+		Exports: integration.Exports{},
+		Build:   integration.New(createIntegration, "process_exporter"),
+	})
+}
+
+func createIntegration(opts component.Options, args component.Arguments) (integrations.Integration, error) {
+	cfg := args.(Config)
+	return cfg.Convert().NewIntegration(opts.Logger)
+}
+
+// DefaultConfig holds the default settings for the process_exporter integration.
+var DefaultConfig = Config{
+	ProcFSPath: "/proc",
+	Children:   true,
+	Threads:    true,
+	SMaps:      true,
+	Recheck:    false,
+}
+
+// Config is the base config for this component.
+type Config struct {
+	ProcessExporter MatcherRules `river:"process_names,block,optional"`
```

Comment:
Should this be `ProcessExporter []MatcherGroup`? This might change the river config a bit so I'll defer to what you think makes sense for this wrapper.